### DISCUSSION TITLE
Turning off taildwind not purging still shows warn messages in the console

### DISCUSSION THREAD

**Discussion by raviSussol** [tailwindlabs/tailwindcss#5073] [Help]
I've setup purging tailwind unused css via webpack with postcss-purgecss library and it successfully purges all unused tailwind css but the warn message `Tailwind is not purging unused styles because no template paths have been provided...` always displays in the console.

```
warn - Tailwind is not purging unused styles because no template paths have been provided.
warn - If you have manually configured PurgeCSS outside of Tailwind or are deliberately not removing unused styles, set `purge: false` in your Tailwind config file to silence this warning.
warn - https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
```

How to turn off that warn message from the console?

I've already tried deleting `purge` option from the `tailwind.config.js` file - still showing up. Tried with `purge: { enabled: false }` too - not working... Is there any settings need to be done to turn off that warning message in the console? Thanks.

**Comment by simonswiss** [comment]
Hey!

What version of Tailwind are you using? Do you mind sharing a repo of this in a little code repo?

If you were to remove the `purge` option, the normal behaviour should be to not display these messages.

Have you by any chance named your config file differently than `tailwind.config.js`? If that's the case, you might be compiling from the implicit default config, and any changes in your actual config would be ignored.

If that's the case, you need to specify the path to this config file in your `postcss.config.js` file:

![image](https://user-images.githubusercontent.com/485747/127804586-03fdaefa-adbb-400e-8527-119abb4b8196.png)

**Reply by raviSussol** [reply]
Hi @simonswiss never mind! I fixed it by using `purge: false`option which I tried to find in the docs but couldn't find that so. But just followed what console warning message has to say and it just worked - 😄 so can be closed it. Thanks for the heads up thought.
